src/System.Security.Cryptography.Csp/src/System.Security.Cryptography.Csp.csproj

```diff
@@ -43,11 +45,14 @@
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'net463'">
     <TargetingPackReference Include="mscorlib" />
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(TargetsWindows)' == 'true'">
+    <AdditionalFiles Include="$(ToolsDir)PinvokeAnalyzer_OneCoreApis.txt" />
```

Comment:
I made a fix in buildtools to cause this to get picked up correctly, so you shouldn't need it.  And since you don't seem to be adding a netstandard/netcore/netcoreapp TargetGroup you shouldn't have needed it anyways?